templates/room.hbs

```diff
@@ -130,25 +131,24 @@
                 camera="userHeight: 1.6"
                 personal-space-bubble
                 look-controls
-                networked="template: #head-template; showLocalTemplate: false;" 
+                networked="template: #head-template; showLocalTemplate: false;"
             ></a-entity>
 
             <a-entity
                 id="body"
                 body-controller="eyeNeckOffset: 0 -0.11 0.09; neckHeight: 0.05"
                 networked="template: #body-template;"
             ></a-entity>
-             
             <a-entity
                 id="nametag"
                 networked="template: #nametag-template; showLocalTemplate: false;"
             ></a-entity>
 
             <a-entity
                 id="left-hand"
-                split-axis-events
                 hand-controls2="left"
-                axis-dpad="centerZone: 1"
+                tracked-controls
```

Comment:
tracked-controls gets added by hand-controls (via the platform specific controller components), I dont think you need this here.